examples/encfs/README.md

```diff
@@ -122,13 +136,13 @@ At this point, the `encfs-sidecar-args.json` file should be completely filled ou
 
 #### 7. Generate Security Policy
 
-At this point, the `aci-arm-template.json` file should be filled out except for the `ccepolicy` field. After installing the [Azure `confcom` CLI extension](#policy-generation), run the following command to generate the security policy and include the `--debug-mode` option so that the policy allows users to shell into the container. The --debug-mode option is only needed for testing purposes and should not be used in production, as it allows users to shell into the containers.
+Configure the [LogFile](aci-arm-template.json#L39) and [LogLevel](aci-arm-template.json#L43) fields on the `aci-arm-template.json`. eg. `log.txt` as the value of [LogFile] so that users can cat `log.txt` at the root of encrypted-filesystem-sidecar-container container. The default logging is `warning` level. At this point, the `aci-arm-template.json` file should be filled out except for the `ccepolicy` field. Set the value of the `ccePolicy` to an empty string. After installing the [Azure `confcom` CLI extension](#policy-generation), run the following command to generate the security policy and include the `--debug-mode` option so that the policy allows users to shell into the container. The --debug-mode option is only needed for testing purposes and should not be used in production, as it allows users to shell into the containers.
 
 ```
 az confcom acipolicygen -a aci-arm-template.json --debug-mode
 ```
 
-This should  automatically populate the [cce policy](aci-arm-template.json#L106) field of `aci-arm-template.json.` If you run the tool on an arm template with the cce policy field already populated, the tool will prompt you to overwrite the existing policy.
+This should automatically populate the [cce policy](aci-arm-template.json#L106) field of `aci-arm-template.json.` If you run the tool on an arm template with the cce policy field already populated, the tool will prompt you to overwrite the existing policy.

```

Comment:
nit: cce policy should be `ccePolicy` (to be consistent with line 139) and the aci arm template